code/_onclick/hud/hud_datum.dm

```diff
@@ -55,7 +55,7 @@
 	//the screen objects that display mob info (health, alien plasma, etc...)
 	var/list/infodisplay = list()
 	/// /atom/movable/screen/inventory objects, ordered by their slot ID.
-	var/list/inv_slots[SLOT_HUD_AMOUNT]
+	var/list/inv_slots[ITEM_SLOT_AMOUNT]
```

Comment:
This will make a list that is `4194304` null items long. You'll want to change this.

Reply:
Has been adressed, ITEM_SLOT_AMOUNT split up into a bitflag and a number. Any indexes are calculated back from the ITEM_SLOT bitflags to their index via TOBITSHIFT(ITEM_SLOT) +1.